web/pom.xml

```diff
@@ -721,6 +721,54 @@
             </fileset>
           </filesets>
         </configuration>
+        <executions>
+          <execution>
+            <id>reset</id>
+            <goals><goal>clean</goal></goals>
+            <configuration>
+              <filesets>
+                <fileset> <!-- database -->
+                  <directory>.</directory>
+                  <includes>
+                    <include>gn*</include>
```

Comment:
This isn't always `gn`. Maybe it's better to use `${db.name}` here?

Reply:
okay, if that works.

Reply:
ok.